wire-ios-request-strategy/Sources/Request Strategies/Connection/ConnectionRequestStrategyTests.swift

```diff
@@ -247,6 +251,32 @@ class ConnectionRequestStrategyTests: MessagingTestBase {
         }
     }
 
+    func testThatOneOnOneResolverIsInvoked_WhenConnectionRequestIsAccepted() {
+
+        syncMOC.performAndWait {
+            // GIVEN
+            let connection = createConnectionPayload(self.oneToOneConnection, status: .accepted)
+            let eventType = ZMUpdateEvent.eventTypeString(for: Payload.Connection.eventType)!
+            let eventPayload = Payload.UserConnectionEvent(connection: connection, type: eventType)
+            let event = updateEvent(from: eventPayload.payloadData()!)
```

Comment:
I think we should avoid force unwraps in tests too.

Reply:
Done!